src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -22,21 +23,21 @@
     public static class HardwareConstants {
 
         // Color Sensor V3 Constants
-        public static final Color BLUE_CARGO_COLOR = new Color(0.0, 0.4, 0.7019607844);
-        public static final Color RED_CARGO_COLOR = new Color(0.9294117648, 0.1098039216, 0.1411764706);
-        public static final double confidenceThreshold = 0.7;
+        // public static final Color BLUE_CARGO_COLOR = new Color(0.0, 0.4, 0.7019607844);
+        // public static final Color RED_CARGO_COLOR = new Color(0.9294117648, 0.1098039216, 0.1411764706);
+        // public static final double CONFIDENCE_THRESHOLD = 0.7;
 
         // drive can ids are range 1-19 (1 is taken by power distribution module)
         public static final int DRIVETRAIN_FRONT_LEFT_DRIVE_MOTOR = 1, DRIVETRAIN_FRONT_RIGHT_DRIVE_MOTOR = 4,
                 DRIVETRAIN_BACK_LEFT_DRIVE_MOTOR = 7, DRIVETRAIN_BACK_RIGHT_DRIVE_MOTOR = 10;
         public static final int DRIVETRAIN_FRONT_LEFT_ANGLE_MOTOR = 2, DRIVETRAIN_FRONT_RIGHT_ANGLE_MOTOR = 5,
                 DRIVETRAIN_BACK_LEFT_ANGLE_MOTOR = 8, DRIVETRAIN_BACK_RIGHT_ANGLE_MOTOR = 11;
-        public static final int DRIVETRAIN_FRONT_LEFT_ENCODER_PORT = 3, DRIVETRAIN_FRONT_RIGHT_ENCODER_PORT = 6,
-                DRIVETRAIN_BACK_LEFT_ENCODER_PORT = 9, DRIVETRAIN_BACK_RIGHT_ENCODER_PORT = 12;
-        public static final double DRIVETRAIN_FRONT_LEFT_ENCODER_OFFSET = -Math.toRadians(67.852);
-        public static final double DRIVETRAIN_FRONT_RIGHT_ENCODER_OFFSET = -Math.toRadians(221.924);
-        public static final double DRIVETRAIN_BACK_LEFT_ENCODER_OFFSET = -Math.toRadians(214.980);
-        public static final double DRIVETRAIN_BACK_RIGHT_ENCODER_OFFSET = -Math.toRadians(168.398);
+        public static final int DRIVETRAIN_FRONT_LEFT_ENCODER_PORT = -1, DRIVETRAIN_FRONT_RIGHT_ENCODER_PORT = -1,
+                DRIVETRAIN_BACK_LEFT_ENCODER_PORT = -1, DRIVETRAIN_BACK_RIGHT_ENCODER_PORT = -1;
+        public static final double DRIVETRAIN_FRONT_LEFT_ENCODER_OFFSET = 0;// -Math.toRadians(67.852);
+        public static final double DRIVETRAIN_FRONT_RIGHT_ENCODER_OFFSET = 0;// -Math.toRadians(221.924);
+        public static final double DRIVETRAIN_BACK_LEFT_ENCODER_OFFSET = 0;// -Math.toRadians(214.980);
+        public static final double DRIVETRAIN_BACK_RIGHT_ENCODER_OFFSET = 0;// -Math.toRadians(168.398);
```

Comment:
Should either revert to original offsets and encoder ports for the competition or add the ability to swap between the two (similar to what #109 does for gear ratios).